packages/core/src/repository/components-repo.ts

```diff
@@ -59,7 +59,7 @@ export class ComponentsRepo {
 
     constructor(
         public name: string,
-        public getNodeType: GetNode = node => node.type
+        public getNodeType: GetNode = node => node.type as string
```

Comment:
@Fer0x shouldn't this also return a symbol?

Reply:
No, components registered with symbol not supposed to be rendered in natural way via getNodeType/getMatched, only with direct use of `getByType` method.